src/city_score/sources/traillink.py

```diff
@@ -0,0 +1,87 @@
+from ..cache import cache
+from ..decorators import dimension, scorer
+from ..source import Source
+from html.parser import HTMLParser
+import requests
+
+activities_filter = {
+    'Atv': 'ATV',
+    'Bike': 'BIKE',
+    'Birding': 'BIRD',
+    'Cross Country Skiing': 'XSKI',
+    'Dog Walking': 'DOG',
+    'Fishing': 'FISH',
+    'Geocaching': 'GEO',
+    'Hiking': 'HIKE',
+    'Horseback Riding': 'HORSE',
+    'Inline Skating': 'SKTS',
+    'Mountain Biking': 'MTBK',
+    'Running': 'RUN',
+    'Snowmobiling': 'SNOW',
+    'Walking': 'WALK',
+    'Wheelchair Accessible': 'WHEEL'
+}
+
+class TrailLinkMilesParser(HTMLParser):
+    def __init__(self):
+        super().__init__()
+        self.found = False
+        self.total_trail_miles = 0
+
+    def handle_starttag(self, tag, attrs):
+        if tag == 'span':
+            for attr in attrs:
+                if attr[0] == 'itemprop' and attr[1] == 'distance':
+                    self.found = True
+
+    def handle_data(self, data):
+        if self.found:
+            self.total_trail_miles += float(data.split()[0])
+
+    def handle_endtag(self, tag):
+        if tag == 'span' and self.found:
+            self.found = False
+
+class TrailLink(Source):
+    base_url = 'https://www.traillink.com/trailsearch'
+
+def format(number):
```

Comment:
Since [`format`](https://docs.python.org/3.11/library/functions.html#format) is a built-in, can you choose a different name (`format_miles` is 👌🏻)?

Reply:
didn't realize! yea `format_miles` is better